features/product/viewing_products/access_product_show_page_from_products_index.feature

```diff
@@ -0,0 +1,23 @@
+@viewing_products
+Feature: Accessing the product show page from product index
```

Comment:
Since this feature is a bit sophisticated, I'm not sure if two tests with simple assertions are enough to cover the whole scenario.